### DISCUSSION TITLE
Is there a way to trigger a build with a button and not needing to merge? Can't seem to deploy our "deploy" branch after switching it in settings. Keeps deploying the "main" branch.

### DISCUSSION THREAD

**Discussion by bob-zs** [vercel/vercel#8704] [Help] [answered]
Where in the project settings can one trigger a build.

**Comment by amyegan** [comment, marked as answer]
Hi @bob-zs! You can manually [deploy from Vercel CLI](https://vercel.com/docs/cli/deploying-from-cli) or [API](https://vercel.com/docs/rest-api#endpoints/deployments/create-a-new-deployment). If you have set a [production branch](https://vercel.com/blog/custom-production-branch) other than the default, then the next merge or push to that branch will trigger a production deployment.

You can [redeploy existing deployments](https://vercel.com/docs/concepts/deployments/managing-deployments#redeploy-a-project) from the Dashboard. And you have the option to [use the CLI for custom workflows](https://vercel.com/guides/using-vercel-cli-for-custom-workflows), like [GitHub Actions](https://github.com/vercel/examples/tree/main/ci-cd/github-actions), if you need more control over production deployments.

**Comment by bob-zs** [comment]
@amyegan Thanks! Yeah, i can deploy from the cli and with a different branch. Thanks for the info!
<img width="766" alt="Screen Shot 2022-10-11 at 1 53 16 PM" src="https://user-images.githubusercontent.com/4698973/195164507-6cd63cb7-0486-4dba-a321-354912ba98a9.png">
